ga/htdocs/components/95_ga_configs.js

```diff
@@ -737,6 +737,43 @@ Ensembl.GA.eventConfigs.push(
     data            : { url: function() { return this.getURL(window.location.href); } },
     category        : 'SpeciesSelectorLink',
     action          : function () { return this.data.url }
+  },
+
+  // Gene Vatiation table links
+  {
```

Comment:
As discussed, the 3 below can be merged as one with id VariationTableLink, add the url with | , for selector use function() to return based on URL and same for category.

Reply:
That's done.